app/models/entities/sales_order.rb

```diff
@@ -33,6 +40,12 @@ def map_to_connec(entity, organization)
   end
 
   def map_to_external(entity, organization)
+    person_id = entity['person_id']
```

Comment:
references should take care of that too.

idmaps above and below should use helper:
```ruby
idmap = Entities::Item.find_idmap({organization_id: organization.id, external_id: id})
idmap = Entities::Item.find_idmap({organization_id: organization.id, connec_id: id})
```